components/docs/plugins/querySource.tsx

```diff
@@ -13,7 +13,81 @@ const PluginDocsQuerySource = ({
   plugin: string;
   isTablesUpperCase?: boolean;
 }) => {
-  const slug = plugin.toLowerCase();
+  const pluginData = PluginData.find(({ name }) => name === plugin);
+  const table = pluginData.tableAlternative || "table";
+  const column = pluginData.columnAlternative || "column";
+  const queryLanguage =
+    plugin === "Mongo" ? "MongoDB Query Language (MQL)" : "SQL";
+
+  const querySchedule =
```

Comment:
Perhaps these code samples should live on the Plugin model?  All the SQL ones can inherit a common string, and Mongo can provide its own.